docs/aerodromes/classc/Sydney.md

```diff
@@ -296,6 +296,7 @@ The Operational Information field should be updated based on the runway mode and
 | `EXP INSTR APCH`, or<br>`EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL` | `INDEP PARL DEPS IN PROG` |
 | `EXP INDEP VISUAL APCH` | `INDEP PARL APPROACHES AND DEPS IN PROG` |
 | SODROPS | `SIMUL OPP DIR PARL RWY OPS IN PROG` |
+| Curfew Mode Operations | `CURFEW RWY NOMINATION. CURFEW IN OPERATION UNTIL TIME 1900.` |
```

Comment:
This is 2000 during standard time. Perhaps include a note about what this should be during DST vs ST?